http/src/test/scala/filodb/http/PrometheusApiRouteSpec.scala

```diff
@@ -38,19 +38,12 @@ class PrometheusApiRouteSpec extends FunSpec with ScalatestRouteTest with AsyncT
   val prometheusAPIRoute = (new PrometheusApiRoute(cluster.coordinatorActor, settings)).route
 
   private def setupDataset(): Unit = {
-    val command = SetupDataset(FormatConversion.dataset.ref, DatasetResourceSpec(2, 1), noOpSource, TestData.storeConf)
-    probe.send(clusterProxy, command)
-    probe.expectMsg(DatasetVerified)
-  }
-
-  before {
-    probe.send(cluster.coordinatorActor, NodeProtocol.ResetState)
-    probe.expectMsg(NodeProtocol.StateReset)
-    // Note: at this point all ingestor actors are shut down
-    cluster.metaStore.clearAllData().futureValue
-    cluster.metaStore.newDataset(FormatConversion.dataset).futureValue shouldEqual Success
-    probe.send(clusterProxy, NodeProtocol.ResetState)
-    probe.expectMsg(NodeProtocol.StateReset)
+    val command = SetupDataset(FormatConversion.dataset, DatasetResourceSpec(2, 1), noOpSource, TestData.storeConf)
```

Comment:
Again we should just rely on a config file for tests

Reply:
I'll am using config files where ever possible like multi-jvm tests. But where we inject datasets manually, it is harder to inject config. I'll try.